File: src/main/java/com/nbteam/killer/helper/domain/Player.java
package com.nbteam.killer.helper.domain;

import com.nbteam.killer.helper.enums.RoleEnum;
import lombok.Data;

@Data
public class Player {

    public Player(RoleEnum role) {
        this.role = role;
    }

    RoleEnum role;

    Integer number;

}

File: killer-helper-base/src/main/kotlin/com/nbteam/killer/helper/base/domain/Player.kt
package com.nbteam.killer.helper.base.domain

import com.nbteam.killer.helper.base.enums.RoleEnum
import lombok.Data

@Data
open class Player(
    /**
     * 角色
     */
    var role: RoleEnum?
) {
    /**
     * 座位号
     */
    var number: Int = 0

}